rules-reviewed/eap6/websphere/websphere-catchall.windup.xml

```diff
@@ -83,7 +83,7 @@
                         </not>
                     </when>
                     <perform>
-                        <hint title="IBM ILog proprietary type reference" effort="0" category-id="potential">
+                        <hint title="IBM ILog proprietary type reference" effort="1" category-id="potential">
```

Comment:
IMO, I think the potential's should be left to the users to customize. Windup is a rule based toolkit exactly for purpose - so that people can customize what it does. Someone will want 0, someone 32.